moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1874,7 +1841,7 @@ bool RobotState::setFromIKSubgroups(const JointModelGroup* jmg, const EigenSTL::
         return true;
       }
     }
-  }
+  } while ((ros::WallTime::now() - start).toSec() > timeout);
```

Comment:
If we are going to loop then wouldn't we need to set `timeout = timeout - elapsed`?

Reply:
Also, I don't see where it is set `first_seed=false`. Am I missing something?

Reply:
Well spotted. Probably this function was never really used in the past.
I also noticed that the timeout checking was using the wrong comparison operator...